packages/vaul-vue/src/DrawerContent.vue

```diff
@@ -14,6 +15,13 @@ const snapPointHeight = computed(() => {
   return '0'
 })
 
+const handlePointerDownOutside = (event: CustomEvent) => {
```

Comment:
We use `PointerEvent` on all other pointer related functions, any particular reason this is a `CustomEvent`

Reply:
Ah good catch! No, no particular reason will fix that up.